turbinia/processors/archive_test.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the Archive processor to compress and decompress folders."""
+
+from __future__ import unicode_literals
+
+import os
+import tarfile
+import unittest
+
+from random import randint
+from shutil import rmtree
+from turbinia.processors import archive
+from turbinia import TurbiniaException
+
+
+class ArchiveProcessorTest(unittest.TestCase):
+  """Tests for Archive Processor."""
+  filedir = os.path.dirname(os.path.realpath(__file__))
+  testdir = os.path.join(filedir, '..', '..', 'test_data', "archive_test")
```

Comment:
Can we use a temporary directory instead of something relative to the code base?  Look in other tests for calls to `tempfile.mkdtemp()`.